interface/modules/custom_modules/oe-module-faxsms/setup_email.php

```diff
@@ -0,0 +1,160 @@
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+require_once(__DIR__ . "/../../../globals.php");
+
+use OpenEMR\Core\Header;
+use OpenEMR\Modules\FaxSMS\Controller\EmailClient;
+
+$emailSetup = new EmailClient();
+$credentials = $emailSetup->getEmailSetup();
+
+if ($_SERVER['REQUEST_METHOD'] === 'POST') {
+    $credentials = array(
+        'sender_name' => $_POST['sender_name'],
+        'sender_email' => $_POST['sender_email'],
+        'notification_email' => $_POST['notification_email'],
+        'email_transport' => $_POST['email_transport'],
+        'smtp_host' => $_POST['smtp_host'],
+        'smtp_port' => $_POST['smtp_port'],
+        'smtp_user' => $_POST['smtp_user'],
+        'smtp_password' => $_POST['smtp_password'],
+        'smtp_security' => $_POST['smtp_security'],
+        'notification_hours' => $_POST['notification_hours'],
+        'smsMessage' => $_POST['smsmessage']
+    );
+    $emailSetup->saveEmailSetup($credentials);
+}
+
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <title><?php echo xlt("Patient Reminder Setup") ?></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <?php Header::setupHeader(); ?>
+    <script>
+        const popNotify = function (e, ppath) {
+            try {
+                top.restoreSession();
+            } catch (error) {
+                console.log('Session restore failed!');
+            }
+            let msg = <?php echo xlj('Are you sure you wish to send all scheduled reminders now?') ?>;
+            if (e === 'live') {
+                let yn = confirm(msg);
+                if (!yn) {
+                    return false;
+                }
+            }
+            let msg1 = <?php echo xlj('Appointment Reminder Alerts!') ?>;
+            dlgopen(ppath, '_blank', 1240, 900, true, msg1);
+        };
+    </script>
+</head>
+<body>
+    <div class="container">
+        <div class="d-flex justify-content-between align-items-center">
+            <h4><?php echo xlt("Setup Patient Reminder Credentials") ?></h4>
+            <div class="dropdown mr-1">
+                <button class="btn btn-outline-primary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                    <?php echo xlt("Reminder Actions") ?>
+                </button>
+                <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+                    <a class="dropdown-item" href="#" onclick="popNotify('', './library/rc_sms_notification.php?dryrun=1&type=email&site=<?php echo attr_js($_SESSION['site_id']) ?>')"><?php echo xlt('Test Email Reminders'); ?></a>
+                    <a class="dropdown-item" href="#" onclick="popNotify('live', './library/rc_sms_notification.php?type=email&site=<?php echo attr_js($_SESSION['site_id']) ?>')"><?php echo xlt('Send Email Reminders'); ?></a>
```

Comment:
change attr_js to attr_url

Reply:
actually, above 2 are bit complicated, but i think above will suffice. A more complete way may be something like:
```
<a class="dropdown-item" href="#" onclick="popNotify('live', './library/rc_sms_notification.php?type=email&site=' + <?php echo attr_js(urlencode($_SESSION['site_id'])) ?>)"><?php echo xlt('Send Email Reminders'); ?></a>
```